includes/template-functions.php

```diff
@@ -996,8 +991,8 @@ function sensei_the_single_lesson_meta() {
 	// Complete Lesson Logic
 	do_action( 'sensei_complete_lesson' );
 	// Check that the course has been started
-	if ( Sensei()->access_settings()
-		|| Sensei_Utils::user_started_course( $lesson_course_id, get_current_user_id() )
+	if ( ! Sensei()->access_settings()
```

Comment:
I don't understand this change. Admittedly, this function seems a bit confusing.

If my understanding is correct, `access_settings()` checks to see whether the user has general permission to view any content (e.g. if they are an admin, or if the "Access Permissions" setting allows content to be viewed by not-logged-in users). So I think it makes sense that we would check to see if it is `true` in this scenario, not `false`.

Does that sound right?

Reply:
Yeah, sorry about that, I misread this after digging into access_settings() in an other part. Reverting in a41bed2